book/website/afterword/glossary.md

```diff
@@ -170,14 +177,23 @@ Digital Object Identifier
 
 ```{glossary}
 
+End to End Test
+  A test that runs the program from beginning to end and verifies that the output is correct.
+
 Epistemology
-  Theory of knowledge and deals with how knowledge is gathered and from which sources. In research terms your view of the world and of knowledge strongly influences your interpretation of data and therefore your philosophical standpoint should be made clear from the beginning. (Source: [Post by Nicole Brown](https://www.nicole-brown.co.uk/epistemology/))
+  Theory of knowledge and deals with how knowledge is gathered and from which sources.
+  In research terms your view of the world and of knowledge strongly influences your interpretation of data and therefore your philosophical standpoint should be made clear from the beginning.
+  (Source: [Post by Nicole Brown](https://www.nicole-brown.co.uk/epistemology/))
 
 Equitable, Diverse and Inclusive Practices
   Ensuring scholarship is open to anyone without barriers based on factors such as race, background, gender, and sexual orientation.
 
-End to End Test
-  A test that runs the program from beginning to end and verifies that the output is correct.
+Ethical Source Software
+  These licenses were created as 'traditional' {term}`open source <Open Source Software>` licenses make no restictions on the uses to which  the software can be put including unethical uses.
+  These licenses require that software is used in ways that respect fundamental human rights.
+  Whilst these licenses sought to be considered {term}`open source <Open Source Software>` their terms place restictions on the uses to which the software can be put so that anyone using the software in a way the licenses consider unethical would potentially expose themselves to liability.
```

Comment:
```suggestion
  Whilst these licenses sought to be considered {term}`open source <Open Source Software>`, their terms place restrictions on the uses to which the software can be put so that anyone using the software in a way the licenses consider unethical would potentially expose themselves to liability.
```